README.md

```diff
@@ -43,32 +43,34 @@ for definition of maturity levels).
 
 Components marked `Stable` provide the following guarantees:
 
-- Field types will not change.
-- Field numbers will not change.
-- Numbers assigned to enum choices will not change.
-- Service names and service package names will not change.
-- Service operation names, parameter and return types will not change.
+- Field types, numbers and names will not change.
+- Names of messages and enums will not change.
```

Comment:
Why? Not important at all for wire compatibility? Only generated code. I think we should say that "SIGs" must not expose this externally.

Not that I want to do this, but I want to clarify the "boundaries" and this is only related to generated code.

Reply:
Yes. Not important from wire perspective.

However, in this PR I am going for full symbolical compatibility of the generate code (except stability of the generator itself, which we don't guarantee of course).